clients/src/main/java/org/apache/kafka/clients/admin/AdminClientConfig.java

```diff
@@ -174,6 +175,11 @@
                                         Importance.MEDIUM,
                                         CLIENT_DNS_LOOKUP_DOC)
                                 // security support
+                                .define(SecurityConfig.SECURITY_PROVIDERS_CONFIG,
```

Comment:
Do we need follow same pattern as other properties to define a new property which refers to SECURITY_PROTOCOL_CONFIG?
Example: 
public static final String SECURITY_PROVDERS_CONFIG = SecurityConfig.SECURITY_PROVIDERS_CONFIG;